### PR TITLE
[ISSUE #4728] Deploy eventmesh-operator on k8s cluster.

### LINKED ISSUE #4728: [Enhancement] Deploy eventmesh-operator to a k8s cluster
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/eventmesh/issues?q=is%3Aissue) and found no similar issues.

### Enhancement Request

Deploy eventmesh-operator to a k8s cluster to automate management.

### Describe the solution you'd like

Deploy eventmesh-operator to a k8s cluster to automate management.

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [ ] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct) *

### PR DESCRIPTION
**Pull request by Alonexc**
Fixes #4728 .

### Motivation

Implementation of deploying eventmesh-operator to a k8s cluster to deploy eventmesh components with one click.

### Modifications

Implementation of deploying eventmesh-operator to a k8s cluster to deploy eventmesh components with one click.

### Documentation

- Does this pull request introduce a new feature? no

### CODE REVIEW

File: eventmesh-operator/config/samples/eventmesh_operator.yaml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+              port: 8081
+            initialDelaySeconds: 5
+            periodSeconds: 10
+          resources:
+            limits:
+              cpu: 500m
+              memory: 128Mi
+            requests:
+              cpu: 10m
+              memory: 64Mi
```
Comment (Pil0tXia) [inline eventmesh-operator/config/samples/eventmesh_operator.yaml:63]: Is that enough?

Comment (Alonexc) [reply eventmesh-operator/config/samples/eventmesh_operator.yaml:63]: @Pil0tXia On my computer can run normally, in use can be modified according to their own needs.

Comment (Pil0tXia) [reply eventmesh-operator/config/samples/eventmesh_operator.yaml:63]: We assume a standard deployment scenario with 3 EventMesh Runtimes and 10 pairs of connectors in the cluster. Users launch 5 pairs of connectors within five minutes from the console. We would like the default configuration to meet the requirements of this scenario.

Comment (Alonexc) [reply eventmesh-operator/config/samples/eventmesh_operator.yaml]: image
The test has been modified, as shown in the figure, and the deployment can be basically completed in one minute.

**Review by Pil0tXia — APPROVED** [review]

**Review by xwm1992 — APPROVED** [review]

**Pull request merged**